pymoab-py2-18.04

```diff
@@ -0,0 +1,57 @@
+FROM  ubuntu:18.04
+
+# set timezone
+ENV HOME /root
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+
+RUN apt-get -y --force-yes update
+RUN apt-get install -y --force-yes \
+    software-properties-common \
+    build-essential \
+    git \
+    cmake \
+    gfortran \
+    libblas-dev \
+    python-pip \
+    liblapack-dev \
+    libhdf5-dev \
+    autoconf \
+    libtool
+
+# need to put libhdf5.so on LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/lib/x86_64-linux-gnu
+ENV LIBRARY_PATH /usr/lib/x86_64-linux-gnu
+
+# upgrade pip and install python dependencies
+ENV PATH $HOME/.local/bin:$PATH
+RUN python -m pip install --upgrade pip
+RUN pip install numpy scipy cython tables matplotlib setuptools future pytest
+
+# make starting directory
+RUN cd $HOME \
+  && mkdir opt
+
+# build MOAB
+RUN cd $HOME/opt \
+    && mkdir moab \
+    && cd moab \
+    && git clone https://bitbucket.org/fathomteam/moab \
+    && cd moab \
+    && git checkout -b Version5.1.0 origin/Version5.1.0 \
+    && autoreconf -fi \
+    && cd .. \
+    && mkdir build \
+    && cd build \
+    && ../moab/configure --enable-shared --enable-optimize --enable-pymoab --disable-debug --with-hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial --prefix=$HOME/opt/moab \
```

Comment:
I would really like to switch to using CMake for all of our reference builds

Reply:
I don't usually build w/ CMake, but is this what you're asking for? (with `-DENABLE_PYMOAB=ON`)
https://github.com/CNERG/dockerfiles/blob/master/moab#L21